API/Controllers/TournamentsController.cs

```diff
@@ -15,17 +15,33 @@ namespace API.Controllers;
 public class TournamentsController(ITournamentsService tournamentsService) : Controller
 {
     /// <summary>
-    /// List all tournaments
+    /// Get all tournaments which fit an optional request query
     /// </summary>
+    /// <param name="requestQuery">The optional request query filter</param>
     /// <remarks>Will not include match data</remarks>
-    /// <response code="200">Returns all tournaments</response>
+    /// <response code="200">Returns all tournaments which fit the request query</response>
+    /// <response code="401">If a request is made to fetch all tournaments without
+    /// the system claim</response>
```

Comment:
Move opening and closing xml tags to their own line

Reply:
This should not be a system-only endpoint it should be a user/client endpoint

Reply:
401 Does not need to be documented here, it is implicit because of the authorize attribute